utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -206,3 +206,13 @@ annotation class EnableSpotlightUi
 
 /** Default value for the feature flag corresponding to [EnableSpotlightUi]. */
 const val ENABLE_SPOTLIGHT_UI_DEFAULT_VALUE = true
+
+/**
+ * Qualifier for the platform parameter that controls whether configuration change for Input
+ * Interactions.
+ */
+@Qualifier
+annotation class EnableConfigurationChange
```

Comment:
Per my comment above, this name might be a bit generic. Perhaps instead: ``EnableInteractionConfigChangeStateRetention`` might be clearer? Ditto for the default value below & providers above.

Reply:
Updated to `EnableInteractionConfigChangeStateRetention`